resp/README.md

```diff
@@ -4,203 +4,23 @@ The goal of the `resp` directory is to build a `golang` source module that allow
 response information discovery for downstream programs but primarily to build `stationxml` representations
 of the meta information.
 
-### helper files
+### ResponseType
 
-The main `golang` output is the automatically built `auto.go` file. There are some helper functions and
-structure definitions in the `response.go` and `streams.go` files. The later provides
-a mechanism to discover all possibly configured streams for a datalogger and sensor pair.
+The _ResponseType_ is a struct that mimics the _StationXML ResponseType_ element, but is not constrained to
+any particular version. For this reason it can be used to load base XML files which can then be used to
+construct the full response needed for _StationXML_.
 
-### generate
+The location of the base XML files can be found in:
 
-The code needed to build the `auto.go` file using the raw configuration files can be found in the
-`generate` sub-directory. This code is usually run in the main `resp` directory via a call to
-`go generate`.
+- `auto` -- where the original response files have been converted into an XML equivalent.
+- `files` -- where response files have been handcrafted as needed.
+- `nrl` -- where responses derived from the NRL (notional response library) can be stored.
```

Comment:
minor comment, I think the name is
"Nominal Response Library"

Reply:
good catch, have updated.

The actual XML is generated by other code so you're checking the difference between the two services and how the decode/re-encode the XML (some optional features have been dropped as the were actually removed in later versions).